utils/wasm-instrument/src/tests.rs

```diff
@@ -85,85 +83,6 @@ fn prebuilt_simple_module() -> elements::Module {
         .build()
 }
 
-#[test]
```

Comment:
better maybe to replace with test that instrumenting with memory.grow instruction leads to error / panic ?

Reply:
done in f95847462c6b893985d8bcac398b61743862394f